ui-tests/src/test/java/io/syndesis/qe/steps/connections/ConnectionSteps.java

```diff
@@ -17,108 +16,63 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
-import java.util.Optional;
 
 import cucumber.api.java.en.Then;
 import cucumber.api.java.en.When;
-import io.syndesis.qe.accounts.Account;
-import io.syndesis.qe.accounts.AccountsDirectory;
-import io.syndesis.qe.pages.connections.detail.ConnectionDetailPage;
-import io.syndesis.qe.pages.connections.edit.ConnectionCreatePage;
-import io.syndesis.qe.pages.connections.list.ConnectionListPage;
-import io.syndesis.qe.pages.connections.list.ConnectionsListComponent;
+import io.syndesis.qe.fragments.common.menu.KebabMenu;
+import io.syndesis.qe.pages.connections.Connections;
 import lombok.extern.slf4j.Slf4j;
 
-/**
- * Created by sveres on 11/10/17.
- */
 @Slf4j
 public class ConnectionSteps {
 
-    private ConnectionCreatePage editPage = new ConnectionCreatePage();
-    private ConnectionDetailPage detailPage = new ConnectionDetailPage();
-    private ConnectionListPage listPage = new ConnectionListPage();
+    private Connections connectionsPage = new Connections();
 
-    @Then("^Camilla is presented with \"([^\"]*)\" connection details")
-    public void verifyConnectionDetails(String connectionName) {
-        log.info("Connection detail page must show connection name");
-        assertThat(detailPage.connectionName(), is(connectionName));
+    @When("^Camilla selects \"([^\"]*)\" connection$")
+    public void selectConnectionType(String connectionName) {
+        connectionsPage.openConnectionDetail(connectionName);
     }
 
     @Then("^Camilla can see \"(\\w+)\" connection$")
-    public void expectConnectionTitlePresent(String connectionName) {
-        listPage.getListComponent().getConnectionByTitle(connectionName).shouldBe(visible);
+    public void checkConnectionIsPresent(String connectionName) {
+        connectionsPage.getConnection(connectionName).shouldBe(visible);
     }
 
     @Then("^Camilla can not see \"([^\"]*)\" connection anymore$")
-    public void expectConnectionTitleNonPresent(String connectionName) {
-        listPage.getListComponent().getConnectionByTitle(connectionName).shouldBe(not(exist));
+    public void checkConnectionIsNotPresent(String connectionName) {
+        connectionsPage.getConnection(connectionName).shouldNot(exist);
     }
 
-    @Then("^she is presented with a connection create page$")
-    public void editorOpened() {
-        ConnectionCreatePage connPage = new ConnectionCreatePage();
-        connPage.getRootElement();
-    }
-
-    @When("^Camilla deletes the \"([^\"]*)\" connection$")
+    @When("^Camilla deletes \"([^\"]*)\" connection$")
     public void deleteConnection(String connectionName) {
-        ConnectionsListComponent listComponent = new ConnectionsListComponent();
-        listComponent.deleteConnection(connectionName);
-    }
-
-    @When("^Camilla selects the \"([^\"]*)\" connection$")
-    public void selectConnection(String connectionName) {
```

Comment:
Imho we're missing now this method that was meant to select named connection when integration was created.

Reply:
@mastepan this and previous comments re connected. We need to be able to select connection type when creating it and also named connection for creating integration. I guess for now just add back this `selectConnection` method.